src/app/services/signing/signing.service.ts

```diff
@@ -0,0 +1,16 @@
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs/Observable';
+import { AppSettings } from '../../app.settings';
+import { RequestService } from '../request/request.service';
+
+@Injectable()
+export class SigningService {
+  private requestService: RequestService = new RequestService();
+
+  public getUploadInfoAndSignature(): Observable<RequestReponse> {
```

Comment:
I would map the `RequestReponse` type to the type which is really returned in this service. http://rxmarbles.com/#map